docs/sso/local_setup.rst

```diff
@@ -0,0 +1,115 @@
+Local Setup
+===========
+
+.. note::
+    Before you begin, please understand that if you are trying to use SAML
+    authentication from `localhost`, it will likely fail on the round-trip
+    handshake, as it is not a public server.
+
+
+Pre-Requisites
+--------------
+
+First, ensure that you have accounting admin privileges
+(see `add_operations_user` management command)
+
+
+Create a Project
+~~~~~~~~~~~~~~~~
+
+1. Navigate to *+ Add Project*
+2. Project Name: `sparrow`
+3. (Click) **Create Project**
+
+
+Create an Enterprise Software Plan
```

Comment:
I wonder if it is worth moving this `Create an Enterprise Software Plan` section to its own section and referencing it here, as we have found a need for this in other areas outside of SSO right? And having it in a specific location for enterprise setup would make it easier to maintain when, for example, we update the enterprise software plan that should be referenced.

Reply:
maybe a Local Setup sub section is needed (outside of SSO)?

Reply:
Yeah that sounds reasonable. Doesn't have to block this PR,  you or I could move it over in a separate PR after this is merged